syntaxes/vhdl.tmLanguage.yml

```diff
@@ -755,13 +756,13 @@ repository:
           1: { name: keyword.operator.vhdl }
           2: { name: keyword.reserved.vhdl }
           3: { name: entity.name.type.vhdl }
-  
+
   named_association:
     patterns:
       - name: meta.block.named_association.vhdl
-        match: (?i)(?!others)([a-z][a-z0-9_]*)\s*=>
+        match: (?i)(?!others)([a-z][a-z0-9_]*)\s*(=>)
         captures:
-          1: { name: variable.other.member.vhdl }
+          1: { name: variable.parameter.vhdl }
```

Comment:
Usually the named associations are used to pass values to functions/ port maps.
The textmate guidelines, as well as the [sublime](https://www.sublimetext.com/docs/3/scope_naming.html#variable) guidelines mention this scope name for that purpose